src/components/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,21 @@
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+
+type Props = {
+  comments: Comment[] | null;
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  comments && (
```

Comment:
check it in the parent component and decide to render or no this component at all (the same at all similar cases)